core-web/libs/edit-content/src/lib/feature/edit-content/edit-content.layout.component.html

```diff
@@ -1,15 +1,15 @@
-<ng-container *ngIf="formData$ | async as formData; else noContent">
-    <dot-edit-content-form [formData]="formData" (changeValue)="setFormValue($event)" />
-    <aside>
-        <dot-edit-content-aside
-            [contentLet]="formData.contentlet"
-            [contentType]="formData.contentType" />
-        <p-button
-            [label]="'Save' | dm"
-            (click)="saveContent()"
-            data-testId="button-save"></p-button>
+<ng-container *ngIf="vm$ | async as vm; else noContent">
+    <dot-edit-content-toolbar
+        class="header"
+        [actions]="vm.actions"
+        (actionFired)="fireAction($event)"></dot-edit-content-toolbar>
+
+    <dot-edit-content-form class="body" [formData]="vm" (changeValue)="setFormValue($event)" />
+    <aside class="sidebar">
+        <dot-edit-content-aside [contentLet]="vm.contentlet" [contentType]="vm.contentType" />
     </aside>
-    <p *ngIf="isContentSaved">Content saved!</p>
 </ng-container>
 
 <ng-template #noContent>No content to show</ng-template>
+
+<p-toast></p-toast>
```

Comment:
Don't we have this global in the whole app?

Reply:
We have one in the `main-legacy.component.ts`, do you want me to move it tho the main template?